src/components/MainNav/ProfileDropdown/ProfileDropdown.js

```diff
@@ -122,11 +129,20 @@ class ProfileDropdown extends Component {
   }
 
   toggleDropdown() {
+    // Get items after rechecking for item visibility
+    this.userLinks = this.getDropdownMenuLinks();
```

Comment:
Previously, the function to check the visibility of dropdown items was called only once in the component constructor, however, the conditions for displaying could change, so it makes sense to do an additional check when opening the dropdown.